rskj-core/src/main/java/co/rsk/config/RemascConfigFactory.java

```diff
@@ -44,9 +48,24 @@ public RemascConfigFactory(String remascConfigFile) {
     public RemascConfig createRemascConfig(String config) {
         RemascConfig remascConfig;
 
+        logger.info("createRemascConfig config = {} ", config);
+        logger.info("createRemascConfig configPath = {} ", configPath);
+
         try (InputStream is = RemascConfigFactory.class.getClassLoader().getResourceAsStream(this.configPath)) {
-            JsonNode node = JacksonParserUtil.readTree(mapper, is);
+            String sis = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8))
```

Comment:
the config is constant, isn't it? I don't see any reason to read it from `resources` on each `createRemascConfig()` method call. Can this be done once in the ctor?

Reply:
This code is just to add more logs so I can debug better using grafana

Reply:
Addressed